chips/apollo3/src/gpio.rs

```diff
@@ -116,7 +116,7 @@ impl Port<'_> {
         let mut count = 0;
         while irqs != 0 && count < self.pins.len() {
             if (irqs & 0b1) != 0 {
-                self.pins[count].handle_interrupt();
+                self.pins[count + 32].handle_interrupt();
```

Comment:
Can you add a comment explaining where the "32" comes from here?

Reply:
Done!